docs/help-topics/stonesoup-import/stonesoup-import.yml

```diff
@@ -31,31 +31,36 @@
 
     ***
     **Using a code sample**
-    1. To start with a code sample, click **Start with a sample**, then go ahead and explore or test various languages and frameworks.         
-    2. Click any of the options on the **Select a sample** page, then follow the resulting instructions.
-
+    To start with a code sample, in the code sample **Search** field, enter the name of the code you want to work with, or click **Import sample** on any of the options displayed. Follow the resulting instructions to explore or test whatever language and framework you choose.         
+    
 
 - name: stonesoup-import-configure-component
   tags:
   title: Configure components
   content: |-
     We analyze your code, then define and apply deployment settings for each of your application components. Remember, a “component” is an image that we build from source code in a repository. An application is a collection of these components working together.
 
-    Before you create and deploy your application, look over the values that display here. You can rename any component from this view, too. In particular, pay attention to the following fields:
+    Before you create and deploy your application, look over the values and options that display here. You can rename any component from this view, too. 
 
-    - **Target port**: Is it correct? If not, click in the port field to modify it.
-    - **CPUs**, **memory**, and **instances**: Choose how many of each of these you want for your application, depending on your subscription.
-    - **Route**: Enter the public URL for your published application.
-    - **Environment variables**: Define your environment variables to customize how we deploy your application. Usually, you define these in a devfile or Dockerfile. If your application doesn’t contain either of these file types, we use an application analyzer to extract the necessary information from your code.
-    - **Runtime**: Did we detect the right runtime? If so, you don’t have to do anything with this field. If you want to change your runtime, click the **Runtime** menu and make a selection.
-    - **Send pull request**: Select this option to create a custom pipeline in your Git repository.
+    - **Component name**: Is it correct? Click the GitHub link under this field if you want to check your repository URL.
+    - **Runtime**: Did we detect the right one? If so, you should notice that we suggest appropriate deployment options. If you want to change your runtime, expand the menu next to your runtime name and make a selection.
```

Comment:
`you should notice that we suggest appropriate deployment options`

We only recommend at most a single option. But we always provide a full list of options. But not all of those options are considered `suitable`. The wrong option will fail to build.

Reply:
thanks, @christianvogt. @missmesss took over this content for me right before summit so i could crank out similar content for the upstream docs. i've asked Oss to let us know where this PR and its changes/comments stand.